clients/lsp-terraform.el

```diff
@@ -357,9 +396,15 @@ This is a synchronous action."
   "Show terraform modules with focus on it if IGNORE-FOCUS? is nil."
   (interactive)
   (if (require 'lsp-treemacs nil t)
-      (lsp-terraform-ls--show-module-calls ignore-focus?)
+      (lsp-terraform-ls--show-module-calls ignore-focus? (lsp-workspace-root))
     (error "The package lsp-treemacs is not installed")))
 
+(defun lsp-terraform-ls--modules-refresh ()
+  "Refresh terraform modules data."
+  (interactive)
+  (with-current-buffer lsp-tf--modules-control-buffer
```

Comment:
I guess you need a check if the buffer is alive.

Reply:
Good point, have pushed a fix addressing it. Thanks!